### ISSUE TITLE
dist/curl.min.js does not export window.curl

### ISSUE THREAD

**Issue by karlbowm** [iotaledger/curl.lib.js#15]
as title describes, tested with latest version 3a9f7bd53c27c6cd6bfc60332c8c2c5198e4a081  but also present in earlier releases

**Comment by paulhandy** [comment]
Exactly, I removed that so that it would be included by `require` rather than assumed a global. curl.lib.js is in npm, but I suppose that perhaps it could build curl.min.js from a separate file

**Comment by karlbowm** [comment]
Interesting,  I expected the file to be packaged for browser use. 

Edit: On that not the readme should be updated to reflect that.

**Comment by pRizz** [comment]
@puffski Here is what I had to do to get it working:

```
npm install -g webpack
mv webpack.config.js webpack.config.js.bak # ignore config in current directory
webpack src/index.js curl.min.js # creates curl.min.js that attaches to the browser window object
```
Another snag is that you have to call curl.init() before using it. I think this was a recent addition.
Another issue is that I get an error about `invalid transaction hash` when interacting with the IOTA nodes. Maybe there is a bug here.

**Comment by karlbowm** [comment]
Yeah I have gotten to that point as well now as well, havent managed to fix the issue with invalid transaction hashes either
